src/Utils.ts

```diff
@@ -32,6 +53,46 @@ export function boolString(bool: boolean): string {
   return bool ? chalk.green('good'.padEnd(14)) : chalk.red('wrong'.padEnd(14));
 }
 
+export async function getPastEventsForHub(
+  parameters: ManagerEventParameters,
+  { fromBlock, toBlock }: PastEventOptions,
+  names: ManagerEvent[] = DEFAULT_MANAGER_EVENTS
+): Promise<Array<TypedEvent>> {
+  const provider = getProvider();
+
+  const relyHubAddress = config.get<string>('contracts.relayHubAddress');
+
+  const relayHub = RelayHub__factory.connect(relyHubAddress, provider);
+
+  const eventFilters = await Promise.all(
+    names.map((name) => {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      const filter = relayHub.filters[name](...parameters);
+
+      return relayHub.queryFilter(filter, fromBlock, toBlock);
+    })
+  );
+
+  return eventFilters.flat();
+}
+
+export async function getRelayInfo(
+  relayManagers: Set<string>
+): Promise<IRelayHub.RelayManagerDataStruct[]> {
+  const provider = getProvider();
+
+  const relyHubAddress = config.get<string>('contracts.relayHubAddress');
+
+  const relayHub = RelayHub__factory.connect(relyHubAddress, provider);
```

Comment:
we can have a function that gets the relayhub instance and this function would be used in `getPastEventsForHub` also

Reply:
yeah we use the relayHub on multiple places, but I was thinking about having its own class or its own utility file but Im not pretty sure which approach to use. If you check the relayHub its build in many points of the server.

Reply:
Yes, but in this specific case we do exactly the same operations:
1. get the relayHub address from config
2. get the contract instance.
Hence, I was thinking about something like `getRelayHubFromConfig` (or any other better name)

Reply:
updated